fs/fshttp/http.go

```diff
@@ -61,7 +62,13 @@ func NewTransportCustom(ctx context.Context, customize func(*http.Transport)) ht
 	}
 
 	// Load client certs
-	if ci.ClientCert != "" || ci.ClientKey != "" {
+	if ci.WinCryptAPI {
```

Comment:
Are the wincrypt certs mutually exclusive to the ones supplied on the command line?

Reply:
It's a oversight. Thanks for pointing out.